src/main/scala/com/databricks/spark/sql/perf/Benchmark.scala

```diff
@@ -297,13 +297,13 @@ object Benchmark {
       sqlContext: SQLContext,
       allTables: Seq[Table],
       currentConfiguration: BenchmarkConfiguration,
-      forkThread: Boolean = true) {
+      forkThread: Boolean = true) extends Logging {
     val currentResults = new collection.mutable.ArrayBuffer[BenchmarkResult]()
     val currentRuns = new collection.mutable.ArrayBuffer[ExperimentRun]()
     val currentMessages = new collection.mutable.ArrayBuffer[String]()
 
     def logMessage(msg: String) = {
-      println(msg)
+      logger.info(msg)
```

Comment:
Hmm... when running in Databricks, having it as a println helps me to see what's going on with messages appearing printed out straight under the notebook cell, and not having to look for them in the Driver logs, which will be cluttered with other stuff.